app/src/main/res/drawable/search_widget_favorites_dark_background.xml

```diff
@@ -15,6 +15,6 @@
   -->
 
 <shape xmlns:android="http://schemas.android.com/apk/res/android">
-    <solid android:color="@color/newBlack" />
+    <solid android:color="@color/gray90" />
     <corners android:radius="20dp" />
```

Comment:
is this 20dp? can we use an already set radius?

Reply:
We agreed to change it to 16dp 👍